thewalrus/quantum/means_and_variances.py

```diff
@@ -0,0 +1,306 @@
+# Copyright 2019-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Functions for constructing/calculating the means, variances and covariances of
+Gaussian states.
```

Comment:
Perhaps a better name for this module is `gaussian_statistics.py`? with statistics implying means, variances, covariances, etc.

Reply:
The problem with that name is that Gaussian statistics in the lay world simple means normally distributed things. The GBS distribution is anything but normal.

Reply:
Hmm... `gaussian_state_statistics.py`?